lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -66,3 +65,25 @@
   padding: $nx-spacing-xl $nx-spacing-l $nx-spacing-md $nx-spacing-l;
   white-space: normal;
 }
+
+.nx-body--page-scrolling {
+  overflow: auto;
+
+  .nx-page {
+    display: block;
```

Comment:
> This may be unnecessary in conjunction with my comment about `padding-left` above.

Yup: https://github.com/sonatype/sonatype-react-shared-components/pull/38/commits/ee435457d08da945d632f84bd7e7b815b8f15c80